doc/manpages/man1/qrest.1

```diff
@@ -11,7 +11,7 @@ There are 2 ways to entered the tempo value :
 .IP \(bu 2
 Keyboard numerical input : User simply types tempo value in a text field and hit \'enter\'.
 .IP \(bu 2
-Realtime button click : User clicks a button in sync with beats, and qrest will tell her if she taps steadily.
+Realtime button click : User clicks a button in sync with beats, and qrest will tell her/him if she/he taps steadily.
```

Comment:
Ah ok, a feminism VS machismo stuff. So, I really don't care :)
I took it as a mistake first, but if it's done on purpose. Then, I reversed it. And this PR is all good to be merged in.